README.md

```diff
@@ -1,21 +1,24 @@
-evopedia-html5
+kiwix-html5
 ==============
 
-Evopedia is an offline wikipedia viewer. See the official site : http://www.evopedia.info/
+Kiwix is an offline wikipedia viewer. See the official site : http://www.kiwix.org/
 This application is developed in HTML5/Javascript, with Firefox OS as the primary target.
-It also works on Android (since Firefox 34) but suffers from a major bug : https://bugzilla.mozilla.org/show_bug.cgi?id=1117136.
+It also works on Android (since Firefox 34) but suffers from a major bug of the platform : https://bugzilla.mozilla.org/show_bug.cgi?id=1117136.
 
-You can install the latest stable version from the Mozilla Marketplace : https://marketplace.firefox.com/app/evopedia/
+It's not yet on the Firefox Marketplace, but should be soon.
```

Comment:
In my opinion, we need to fix the bug on big ZIM files first (#156).
I suppose it might take some time, so maybe it would be worth mentioning the Evopedia application on the Marketplace